source/includes/_tipser-react-elements.md

```diff
@@ -0,0 +1,263 @@
+# Tipser React Elements
+Tipser Elements is a React components library from Tipser enabling you to add shoppable content to your site with minimal effort.  You can add simple components like products with buy buttons, collections of products and articles, but also more complex grids or whole pages.
```

Comment:
Maybe "to your site" -> "to your React site"? Or maybe we could rephrase the whole sentence to include the phrase "if your side is powered by / based on React", to be fully clear that this technology is only suitable for _some_ customers with specific tech?